joomla/joomla-cms
# [4.3] Migrate quickicon downloadkey plugin to services

### Summary of Changes

Migrate the quickicon downloadkey plugin to service providers and use SubscriberInterface.

### Testing Instructions

- Install an extension with providing a download key
- Check if quickicons are still working

![image](https://user-images.githubusercontent.com/66922325/200318253-2373db78-d5e6-46a9-9ce9-d2de9e046775.png)
![image](https://user-images.githubusercontent.com/66922325/200318234-e0a3c83d-3a00-4a16-86e9-878d8a7355e4.png)

### Actual result BEFORE applying this Pull Request

Downloadkey quickicon is working

### Expected result AFTER applying this Pull Request

Downloadkey quickicon is working (and the other quickicons are still displayed)

### Link to documentations
Please select:
